### ISSUE TITLE
doesnt open openshell start menu when i click start

### ISSUE THREAD

**Issue by jwnrz** [dremin/RetroBar#625]
it was working for a while until it just stopped i uninstalled both and then reinstalled but its not working for some reason. openshell works when retrobar is not running.

**Comment by whisperer121** [comment]
I've encountered this issue too, the issue is that windows 11 update KB5027303 broke some things and now open-shell won't work with retrobar on windows 1

**Comment by DrBrad22** [comment]
The same has happened with me.

**Comment by tsumanne** [comment]
yep the same things happen to me, but the bug only appear when i have multiple monitor. when its only on one monitor setup its works

**Comment by dremin** [comment]
Have you tried the latest pre-release OpenShell (4.4.189)? For me it solves these problems. https://github.com/Open-Shell/Open-Shell-Menu/releases

**Comment by dremin** [comment]
Closing due to no response.
